BaseTools/Plugin/OverrideValidation/ReadMe.md

```diff
@@ -0,0 +1,176 @@
+# Override Validation Plugin
+
+Module Level Override Validation Plugin and Linkage Creation Command Line Tool
+
+## About
+
+OverrideValidation is a UEFI Build Plugin and Command Line Tool used to create
+linkage between overriding and overridden modules and parse INF files referenced
+in platform DSC files during build process and then produce a TXT report of the
```

Comment:
Similarly, if "TXT" could just be replaced with "text" that would also be better since "TXT" is a popular technology and the "TXT" abbreviation is really unnecessary.

Reply:
I've updated from TXT -> text.